tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -1467,6 +1467,81 @@ public function test_intent_error_throws_exception() {
+
+		$exception = null;
+		try {
+			$this->mock_gateway->process_order_for_confirmed_intent( $order, $payment_intent_id, false );
+		} catch ( WC_Stripe_Exception $e ) {
+			$exception = $e;
+		}
+
+		$this->assertNotNull( $exception, 'Expected WC_Stripe_Exception to be thrown' );
+		$this->assertNotInstanceOf( WC_Stripe_Payment_Cancelled_Exception::class, $exception );
+		$this->assertMatchesRegularExpression( '/not able to process this payment\./', $exception->getMessage() );
+	}
```

PHPMD (2.15.0)

[error] The method test_intent_hard_error_throws_generic_exception is not named in camelCase. (undefined)

(CamelCaseMethodName)